### DISCUSSION TITLE
What is the process for updating Supabase for local dev?

### DISCUSSION THREAD

**Discussion by tholder** [supabase/supabase#14110] [Questions]
I followed the local dev setup and it's working great, however I can see there has been an update to the docker image https://github.com/supabase/supabase/pkgs/container/studio

How do I upgrade my local dev? I tried just a download of that docker image with:

```bash
docker pull ghcr.io/supabase/studio:20230502-d8c0723
```

It pulled it, but I don't think it's running it. I've started and stopped Supabase with the cli.

Thanks

**Comment by magick93** [comment]
Depending on how you installed the cli, you could, for example, run `brew upgrade supabase`. Then `supabase start` will upgrade the images.
